autogpt_platform/backend/backend/util/service.py

```diff
@@ -119,6 +120,12 @@ class RemoteCallError(BaseModel):
 
 class AppService(BaseAppService, ABC):
     fastapi_app: FastAPI
+    log_level: str = "info"
+
+    def set_log_level(self, log_level: str):
+        """Set the uvicorn log level. Returns self for chaining."""
+        self.log_level = log_level
+        return self
```

Comment:
this works but would make more sense as an `__init__` argument imo

Reply:
Adding it as an init would propagate that requirement for all the other services extending it, the implementor has to mention the param on their __init__